docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<p>Great job in providing a clear plot!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>